README.md

```diff
@@ -84,6 +84,35 @@ changelogFile: CHANGELOG.md
 
 When a package is modified in any way (including updates of the context information in the `.janno` file), then the `packageVersion` field SHOULD be incremented and the `lastModified` field updated to the current date.
 
+#### Package versioning
+
+The `packageVersion` field is a mandatory entry of the `POSEIDON.yml` file. It denotes the version of the individual package, using a three-component versioning system derived from [semantic versioning](https://semver.org).
+
+Each version number is comprised of three numbers, separated by a `.`. For example: `0.1.0`, `1.0.0` or `2.1.3`. The first number gives the `major`, the second the `minor` and the third the `patch` component of the version number. For a Poseidon package these components SHOULD be incremented, so raised by one, when the following changes occur:
```

Comment:
```suggestion
Each version number is comprised of three numbers, separated by a `.`. For example: `0.1.0`, `1.0.0` or `2.1.3`. The first number gives the `major`, the second the `minor` and the third the `patch` component of the version number. For a Poseidon package these components SHOULD be incremented, when the following changes occur:
```
A bit pedantic but: incrementing is not simply raising by one, but also zeroing the more minor components. I think the text might therefore mislead users without the knowledge of what that means (who are the exact users that clarification is for). I would instead advocate for showing an example of a version bump alongside the bullet point list.